README.md

```diff
@@ -104,42 +124,53 @@ As a starting point, we recommend [EventStorming](https://www.eventstorming.com/
 ![EventStorming](resources/event_storming.jpeg)
 
 #### Tools
+
 - [Domain Storytelling](https://domainstorytelling.org/)
 - [Example Mapping](https://cucumber.io/blog/bdd/example-mapping-introduction/)
 - [EventStorming](https://www.eventstorming.com/)
 - [User Journey Mapping](https://boagworld.com/audio/customer-journey-mapping/)
 
 #### Who to Involve
+
 - People who design, build, test software
 - People who have domain knowledge
 - People who understand product and business strategy
 - People who understand the customers' needs and problems
 - Real end users
 
 ### 3. Decompose
+
 Decompose the domain into sub-domains - loosely-coupled parts of the domain.
 
 We decompose a large problem domain into sub-domains for a few key reasons:
+
 - reduced cognitive load, so that we can reason about parts of the domain independently,
 - give development teams autonomy, so that they can work on separate parts of the solution,
 - identifying loose-coupling and high-cohesion in the domain which carries over to our software architecture and team structure.
 
+<<<<<<< HEAD
+As a starting point, we recommend [carving up your event storm](resources/event-storm-bounded-context.jpg) into sub-domains and [Context Maps](https://speakerdeck.com/mploed/visualizing-sociotechnical-architectures-with-context-maps).
+=======
 As a starting point, we recommend carving up your event storm into sub-domains and [Context Maps](https://speakerdeck.com/mploed/visualizing-sociotechnical-architectures-with-context-maps?slide=33).
+>>>>>>> upstream/master
```

Comment:
This needs fixing.

Reply:
Done @cakper, forgot to add the actual change I made on previous commit :p